tests/ui/cast/cast-rfc0401-vtable-kinds.rs

```diff
@@ -1,4 +1,8 @@
 //@ run-pass
+
+#![feature(rustc_attrs)]
+#![rustc_no_implicit_bounds]
+
```

Comment:
for unsize coercions/trait object coercions test, the attribute makes them a bit confusing/harder to read for me :thinking:

so maybe only limit the attribute to tests for the trait solver :thinking: